src/cli.rs

```diff
@@ -27,6 +27,10 @@ pub enum Commands {
     ///
     /// Disable options you've enabled through this utility
     Disable(DisableCommand),
+
+    #[clap(subcommand)]
+    /// Switch to alternate kernels
+    SwitchKernel(SwitchKernel),
```

Comment:
Can I get some explanation as to how this will work. What would the command be to enable the xanmod kernel be, for example?

Reply:
```bash
rhino-config switch-kernel xanmod # For xanmod kernel
rhino-config switch-kernel liquorix # For liquorix kernel
```

Reply:
Brilliant, thank you. I like the new feature of the `rhino-config switch-kernel` and how it is kept separately from the other `rhino-config enable/disable` subcommands. My one question is what would happen if someone uses `rhino-config switch-kernel liquorix` and then types `rhino-config switch-kernel xanmod`, will both be installed via `rhino-update` or will it remove one of their config files?

Reply:
Currently if the user runs both of the commands, it'll create both the config files. I can change that if you want.

Reply:
> Currently if the user runs both of the commands, it'll create both the config files. I can change that if you want.

No, I think that works personally. It allows for both kernels to be installed which I assume the user would want. Is there a way to disable these configuration files yet?

Reply:
No, do you want it? If so then write out the command that you would want assigned to that function.

Reply:
Something similar to perhaps `rhino-config disable-kernel xanmod` to disable a kernel that was enabled via switch-kernel

Reply:
how about making it a subcommand like:

To enable xanmod:
```
rhino-config switch-kernel enable xanmod
```

To disable xanmod:
```
rhino-config switch-kernel disable xanmod
```

@MrBeeBenson

Reply:
Yes that's perfect